contributing.md

```diff
@@ -0,0 +1,21 @@
+Unless you explicitly state otherwise, any contribution intentionally submitted
+for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
+dual licensed as above, without any additional terms or conditions
```

Comment:
"above" here still doesn't work because there's no mention about dual-licensing in the text above this line. The "above" is referring to the text that stands above where this text was copied from in README.md

Reply:
I finally figured this out!

I forgot to copy the `LICENSE` section.